clients/banking/src/components/TransferRegularWizardSchedule.tsx

```diff
@@ -74,7 +74,7 @@ export const TransferRegularWizardSchedule = ({
     {
       query: GetIbanValidationDocument,
       variables: {
-        iban: beneficiary.iban,
+        iban: beneficiary.iban.replaceAll(" ", ""),
```

Comment:
can we use `electronicFormat(beneficiary.iban)` from the `"iban"` library instead?